runtime/src/bank.rs

```diff
@@ -949,6 +949,7 @@ impl Bank {
         bank.transaction_processor = TransactionBatchProcessor::new(
             bank.slot,
             bank.epoch,
+            Arc::clone(&bank.feature_set),
```

Comment:
I believe you can write directly `bank.feature_set.clone()`, can't you?

Reply:
You can, yep. I just went with the verbose method to show we're only cloning the `Arc`. We do this in a lot of areas, but it's definitely not required. Your preference!